### PR TITLE
[12.0][MIG] account_move_line_stock_info

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of module to version 12.0

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on local

File: account_move_line_stock_info/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+============================
+Account Move Line Stock Move
```
Comment (rousseldenis) [inline account_move_line_stock_info/README.rst]: @bjeficent I think you have to run oca-gen-addon-readme

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment

File: account_move_line_stock_info/README.rst (hunk 1 above)
Comment (bjeficent) [reply account_move_line_stock_info/README.rst]: You are right, my bad.

File: account_move_line_stock_info/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Move Line Stock Move",
```
Comment (rousseldenis) [inline account_move_line_stock_info/__manifest__.py]: @bjeficent I think the README title is wrong due to this

Comment (bjeficent) [reply account_move_line_stock_info/__manifest__.py]: README updated

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@rousseldenis could you update your review?

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by AaronHForgeFlow** [comment]
@lreficent

**Pull request merged**